apps/site/pages/docs/building-sections/overrides/use-cases/overrides-use-cases-overview.mdx

```diff
@@ -0,0 +1,11 @@
+<header>
```

Comment:
we should create a `_meta.json` file here just to make this docs in order:

1. Override use cases (this overview)
2. Add other icons option to the Alert component
3.  Add an image to your Alert component

<img width="1292" alt="image" src="https://github.com/vtex/faststore/assets/3356699/226537fc-019f-4359-a67c-b90819f714e0">

Reply:
Good point, Fanny! Since this doc will be in the developer portal, I don't think we need to worry about creating the `_meta.json` file here. However, I'm going to include this order there.